jabgui/src/main/java/org/jabref/gui/mergeentries/threewaymerge/FieldRowViewModel.java

```diff
@@ -122,6 +122,8 @@ public FieldRowViewModel(Field field, BibEntry leftEntry, BibEntry rightEntry, B
         });
 
         EasyBind.subscribe(hasEqualLeftAndRightBinding(), this::setIsFieldsMerged);
+
+        autoSelectBetterValue();
```

Comment:
@HoussemNasri is this correct?

Reply:
The auto selection logic was being triggered only when the merge UI was created in FetchAndMergeEntry.

But we use the UI in other places .e.g. DuplicateResolverDialog. I see no harm of generalizing this auto selection in all flows, but we should probably trigger the auto selection from a higher-level location to have more control: ThreeWayMergeView constructor.